### DISCUSSION TITLE
The question of campus location

### DISCUSSION THREAD

**Discussion by lutao2021** [community/community#65807] [GitHub Education]
### Hi everyone,

My verified e-mail is hkust-gz.edu.cn, and I didn't use any VPN. But when I uploaded the admission notice to GitHub, it said that there is no campus location in my country. How can I solve this problem?
Thank you!

•	The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.
•	You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

**Comment by XingHehy** [comment]
I have the same problem. 
Has anyone solved it?

**Comment by elendil7** [comment]
Hello. Please see the following comment for a solution:

https://github.com/orgs/community/discussions/20819#discussioncomment-3282525
